aviary/docs/user_guide/reserve_missions.md

```diff
@@ -5,40 +5,65 @@
 Reserve missions are enabled for the following mission types:
 
 * height_energy    (completed)
-* 2ODF (collocation) (in-progress)
-* 2DOF (shooting)    (future work)
+* 2ODF (collocation) (complete)
+* 2DOF (shooting)    (in-progress)
+* height_energy (shooting)    (ifuture work)
+
+A reserve mission can be created by appending one or more reserve phases to `phase_info` after the last phase of the regular mission. 
+To create a simple reserve mission, copy your current cruise phase which is located in `phase_info`. 
+(Note: You may need to revise some of your assumptions for the copied phase if you are making a reserve phase that is radically different than the original (i.e. original phase was to travel 3000km but reserve phase is 100km)).
+Append that phase to the end of `phase_info`, name it `reserve_cruise` and add `"reserve": True,` to `user_options` for this phase.
+There are two optional flags that can now be added to `user_options`.
+The `"target_duration"` option creates a phase requiring the aircraft to fly for a specific amount of time.
+The `"target_distance"` option creates a phase requiring the aircraft to fly for a specific distance.
+Avoid using the optional flag if you have a reserve phase (i.e climb or descent) where you just want that phase to be completed as quickly as possible.
+The optional flags should not be combined as they will create overlapping constraints creating an infeasible problem.
 
-Reserve missions can be used to create fixed-time or a fixed-distance phases that are appended to the last phase of your nominal mission.
-To do this, set the `"reserve": True,` flag within the `user_options` for your reserve phases within `phase_info`.
-Additionally, if this is a fixed-time phase add `"target_duration"` and if it's a fixed-distance phase add `"target_distance"`.
-Reserve phases must be added to `phase_info` after the regular phases.
-You are not allowed to intersperse regular and reserve phases.
+You can chain together multiple reserve phases to make a complete reserve mission (i.e. climb to altitude, cruise for range, cruise for time, then descend).
+Examples of this are shown in `run_reserve_mission_multiphase.py` and `run_2dof_reserve_mission_multiphase.py`.
 
-The reserve phases will start at the same range and mass as the last regular phases, but the altitudes are not automatically connected.
+The reserve phases will start at the same range and mass as the last regular phases, but all other states (i.e. altitude, mach number) are not automatically connected.
```

Comment:
Maybe "The first reserve phase will start..."  just so they don't think every reserve phase starts at the end of descent.

Reply:
Good idea, I had some other changes and rolled this suggestion into them